app/main/RTMedia.php

```diff
@@ -209,6 +209,17 @@ function set_site_options() {
 		}
 		$this->add_image_sizes();
 		$this->set_image_quality();
+		$this->commentautocomplete( $rtmedia_options, $bp_media_options );
+	}
+
+	public function commentautocomplete( $rtmedia_options = false, $bp_media_options = false ) {
```

Comment:
@wordpressdeepak Add the comment doc block please. Also change the function name to `comment_auto_complete` (Wherever its applicable)